src/main/java/VisualLogic/variables/VSImage.java

```diff
@@ -116,13 +115,13 @@ public void loadFromStream(java.io.FileInputStream fis) {
                 if (image != null) {
                     image.flush();
                 }
-
+                // TODO ignored read here
```

Comment:
Is this still Todo?

Reply:
Yes just wanted to point it out. Sometimes reads from a stream are just ignored.

Reply:
Ok could you then clarify this in the comment a bit more? I though this was a Todo related to this PR.

Reply:
Added comment here 1e91c2fd